### PR TITLE
Phoenix-5797 RVC Offset does not work with tenant views on global indexes

### PR DESCRIPTION
**Pull request by dbwong**

### CODE REVIEW

**Comment by dbwong** [comment]
@yanxinyi @ChinmaySKulkarni review for issue with global indexes in tenant views.

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java [hunk 1]
```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
...
+                        + "    ORGANIZATION_ID CHAR(15) NOT NULL,\n"
+                        + "    PARENT_KEY_PREFIX CHAR(3) NOT NULL,\n"
+                        + "    PARENT_ID CHAR(15) NOT NULL,\n" + "    CREATED_DATE DATE NOT NULL,\n"
+                        + "    DATA VARCHAR   \n" + "    CONSTRAINT PK PRIMARY KEY \n" + "    (\n"
+                        + "        ORGANIZATION_ID, \n" + "        PARENT_KEY_PREFIX,\n"
+                        + "        PARENT_ID,\n" + "        CREATED_DATE\n" + "    )\n"
+                        + ") MULTI_TENANT=true";
+
+        //Index reorders the pk only
+        String
```
Comment (yanxinyi) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: nit: weird line break

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java [hunk 2]
```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
             assertTrue(explainStringBuilder.toString().contains("With RVC Offset"));
         }
     }
+
+    @Test public void testGlobalIndexViewAccess() throws Exception {
+        String
+                ddl =
```
Comment (yanxinyi) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: nit: weird line break

`String ddl =` should be a single line?

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java [hunk 3]
```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
...
+
+        //Index reorders the pk only
+        String
+                indexSyncDDL =
+                "CREATE INDEX IF NOT EXISTS FIELD_HISTORY_ARCHIVE_INDEX\n"
+                        + "ON ARCHIVE.FIELD_HISTORY_ARCHIVE (PARENT_KEY_PREFIX, CREATED_DATE, PARENT_ID)\n"
+                        + "INCLUDE (DATA)";
+
+        String
+                viewDDL =
```
Comment (yanxinyi) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: nit: weird line break

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java [hunk 4]
```diff
@@ -940,4 +941,70 @@ public void testOffsetExplain() throws SQLException {
...
+                "CREATE VIEW IF NOT EXISTS ARCHIVE.\"1HA\" AS SELECT * FROM ARCHIVE.\"FIELD_HISTORY_ARCHIVE\"";
+        try (Statement statement = conn.createStatement()) {
+            statement.execute(ddl);
+            statement.execute(indexSyncDDL);
+            //statement.execute(viewDDL);
```
Comment (yanxinyi) [inline phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: remove unused line

Comment (dbwong) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: Done

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java (hunk 3 above)
Comment (dbwong) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: i think i used the phoenix code formatter and it did this i can see if i can trick it into formatting nicer.

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java (hunk 2 above)
Comment (dbwong) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: i think i used the phoenix code formatter and it did this i can see if i can trick it into formatting nicer.

File: phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java (hunk 1 above)
Comment (dbwong) [reply phoenix-core/src/it/java/org/apache/phoenix/end2end/RowValueConstructorOffsetIT.java]: i think i used the phoenix code formatter and it did this i can see if i can trick it into formatting nicer.

**Comment by yanxinyi** [comment]
can you close this pull request? I already merged your patch, thanks.

**Pull request closed without merging**